--- isolinux.2	2013-11-09 10:42:34.090906457 +0000
+++ isolinux.cfg	2013-11-09 10:43:04.049516395 +0000
@@ -1,4 +1,4 @@
-default kickstart
+default vesa.c32
 timeout 600
 
 display boot.msg
@@ -58,11 +58,6 @@
 menu separator # insert an empty line
 menu separator # insert an empty line
 
-label kickstart
-  menu label ^Kickstart bootstrap node
-  kernel vmlinuz
-  append initrd=initrd.img inst.stage2=hd:LABEL=Fedora\x2019\x20x86_64 text ks=cdrom:/bootstrap-node.ks
-
 label linux
   menu label ^Install Fedora 19
   kernel vmlinuz
